README.md

```diff
@@ -26,9 +206,109 @@ We also use bundling techniques to optimize code delivery and reduce load times
 
 Overall, this technical architecture provides a scalable solution that enables us to add new modules easily while maintaining high stability across multiple platforms.
 
+# Packages
+
+The Unified SDK is a monorepo that contains all the packages from different Immutable products. Each product area has its own package within the monorepo, and these packages are imported and re-exported by one root-level package that serves as a single entry point to access all modules included in the Unified SDK for external consumers.
+
+The following is a list of packages included in the Unified SDK:
```

Comment:
swap all "Unified SDK" wording everywhere to "Typescript SDK"

Reply:
Donezo 👍